utkdigitalinitiatives/polk-correspondence-app
# Ptc 111

**JIRA Ticket**: [https://jira.lib.utk.edu/browse/PTC-111](https://jira.lib.utk.edu/browse/PTC-111)

# What does this Pull Request do?

- Brings the header built in react up to date and ready for responsive testing.
- Incorporates hours from LibCal API.
- Adjusts Polk Hero to not use script fonts.

# How should this be tested?

1. `ant` and upload .xar
2. Make sure header and Polk app open as expected.
3. Script font in hero area should no longer be used. Generic serif font is present.
4. Click *MENU* in top right.
5. LibCal hours should be present.

# Additional Notes:
The pull request largely just brings this repo up to date with development on the universal header that has been completed at [utkdigitalinitiatives/utk_lib_header.](https://github.com/utkdigitalinitiatives/utk_lib_header/tree/www) The general purpose is to provide a checkpoint for responsive and accessibility testing on sparrow.

# Interested parties
@markpbaggett
